azurerm/resource_arm_app_service.go

```diff
@@ -43,7 +43,10 @@ func resourceArmAppService() *schema.Resource {
 							Required:         true,
 							DiffSuppressFunc: ignoreCaseDiffSuppressFunc,
 							ValidateFunc: validation.StringInSlice([]string{
-								"SystemAssigned",
+								string(web.ManagedServiceIdentityTypeNone),
```

Comment:
FYI there seems to be a bug in current version so that when an `identity` block is removed from the configuration, it doesn't update the resource. However setting `type` to `None` makes it possible to disable identities. I didn't make any attempt to fix this.